src/software/logger/network_sink.cpp

```diff
@@ -25,6 +25,10 @@ void NetworkSink::sendToNetwork(g3::LogMessageMover log_entry)
         log_msg_proto->set_line_number(
             static_cast<uint32_t>(std::stoul(log_entry.get().line())));
 
+        TbotsProto::Timestamp timestamp;
+        timestamp.set_epoch_timestamp_seconds(static_cast<double>(time(nullptr)));
```

Comment:
What is this `time()` function? I don't think I've seen it before. 
The approach I've seen us use elsewhere is more similar to this:
https://github.com/UBC-Thunderbots/Software/blob/5691a5ef58fae9ee352528ba97221b63907216d6/src/software/logger/protobuf_sink.cpp#L58-L62

Reply:
it's a function that returns the current time in seconds apparently : https://cplusplus.com/reference/ctime/time/

is the way you suggested how we normally do it? I'll change it